### ISSUE TITLE
create recipe in meal plan

### ISSUE THREAD

**Issue by TWART016** [TandoorRecipes/recipes#1285]
**Is your feature request related to a problem? Please describe.**
Sometimes I just want to note what I am cooking. Therefore I do no have a recipe. Inside meal plan I can search for a recipe but cannot create a new one.

**Describe the solution you'd like**
create a recipe inside meal plan wizward

**Comment by vabene1111** [comment]
but that would make that an empty recipe which would show up in the search, is this really useful ? 

i mean it could be, you would see it and remind yourself of cooking it again and you could easily assign some keywords if you want but it could also be confusing :thinking: 

i would love some additional input on this matter (pros/cons) ?

**Comment by philip-n** [comment]
I'd give my opinion, but I don't think I understand the proposal ;)

Which screen are we talking about? On the meal planner, I can see my navigation bar (I have it pinned) that has a `+` button that leads to the creation of a new recipe. Isn't that the required functionality?

**Comment by vabene1111** [comment]
![image](https://user-images.githubusercontent.com/6819595/149219842-328b9b5e-cf61-4cbd-94e4-723629a812d1.png)
this recipe selection

**Comment by TWART016** [comment]
One use case is the collection of all recipes. Sometimes the recipe is that easy and do not need a recipe with ingredients, steps, time, ...

Sometimes I look on the plan and see one day is missing. Then I have an idea and just want to note it down to reminder the next days.

**Comment by vabene1111** [comment]
thats what the "title" field is for, you can create plan entries without linking recipes to them. Maybe this should be clearer in the UI. But i think your feature request might still be interesting to implement

**Comment by TWART016** [comment]
Yes I can create an entry with just a title and meal type. In this case I have to type it in every time manually. And I want to store this "idea" in a book. Or want to use the Auto-Planner

And this entry cannot be mapped to a meal type #1186

**Comment by vabene1111** [comment]
yes i like those benefits, this will be implemented.
